keyboards/takashicompany/ergomirage/info.json

```diff
@@ -0,0 +1,399 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "Ergomirage",
+    "maintainer": "takashicompany",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "D4",
+            "C6",
+            "D7",
+            "E6",
+            "B4",
+            "B5"
+        ],
+        "rows": [
+            "F4",
+            "F5",
+            "F6",
+            "F7",
+            "B1",
+            "B3",
+            "B2",
+            "B6"
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x7463",
+        "pid": "0x0044"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {
+                    "matrix": [
+                        0,
+                        0
+                    ],
+                    "x": 1,
+                    "y": 1.25
+                },
+                {
+                    "matrix": [
+                        0,
+                        3
+                    ],
+                    "x": 2,
+                    "y": 1.25
+                },
+                {
+                    "matrix": [
+                        1,
+                        0
+                    ],
+                    "x": 0,
+                    "y": -10.5
```

Comment:
`x` and `y` positions cannot be negative. Your layout looks like this:
![image](https://user-images.githubusercontent.com/4781841/236670001-b495e3c8-1ae3-4e48-bbbc-7b793f2be58b.png)

Reply:
I fixed https://github.com/qmk/qmk_firmware/pull/20655/commits/5116eb7aa55a6b7640829baf726e67a0e469f532